fapolicyd.spec

```diff
@@ -8,6 +8,7 @@ Source0: https://people.redhat.com/sgrubb/fapolicyd/%{name}-%{version}.tar.gz
 BuildRequires: kernel-headers
 BuildRequires: systemd-devel libgcrypt-devel rpm-devel file-devel
 BuildRequires: libcap-ng-devel libseccomp-devel lmdb-devel
+Suggests: %{name}-dnf-plugin
```

Comment:
Why is this a suggests? If there is no plugin installed, any update will go unnoticed and fapolicyd will break the system. So, I think its fine to just add this to the main package rather than make it a subpackage.

Reply:
It is a weak dependency. It means that it is installed when fapolicyd is installed just as normal Requires but it can be uninstalled by administrator. In this case he can use only cli tool for that manually.

https://fedoraproject.org/wiki/Packaging:WeakDependencies

Reply:
By default dnf plugin will be installed with fapolicyd.

Reply:
Why would an admin remove the plugin and risk breaking their system?